garden_ai/app/notebook.py

```diff
@@ -218,16 +236,30 @@ def start(
         source_path = top_level_dir / "notebook_templates" / template_file_name
         shutil.copy(source_path, notebook_path)
 
+    # Update garden metadata in notebook
+    _set_notebook_metadata(notebook_path, base_image_uri, requirements_data)
```

Comment:
I had been imagining the base image "name" in the notebook cell, like "3.10-sklearn" or what have you. The base image URI is useful for quickly getting the notebook running again on the same image, but it doesn't accomplish the goal of making it easy to pass over a notebook and have all the information you need to run it in the cell. The base image URI will point to a local image on someone's computer and won't be useful when shared.

This is probably something I didn't think hard enough about when writing the ticket and in our early conversations, that's on me. But I think the visible metadata in the cell shouldn't have the base image URI in it. We _may_ still want to save the base image URI somewhere invisible in the notebook for our own convenience.

This is probably something we should sync on.